app/controllers/case_workers/claims_controller.rb

```diff
@@ -60,7 +56,6 @@ def update
 
   def prepare_show_action
     @claim.assessment = Assessment.new if @claim.assessment.nil?
-    @doc_types = DocType.all
```

Comment:
🗒  There were cases on this controller where we we setting this _unsused_ variable 3 times in the same action! 😰